README.md

```diff
@@ -13,12 +13,16 @@ The script in this repository contains a few additional features which I found u
 ## Usage
 
 ```
-./maildir2gmail.py [-f] [-n] -u <username> -p <password> <maildirs>
+./maildir2gmail.py [-f] [-n] [--ignore-missing-date] [--max-size=MAX_SIZE] -u <username> -p <password> <maildirs>
 ```
 
 By default, all mails are uploaded to _All Mail_ in Gmail, following the Google concept of "never delete mails, just archive everything".
 A folder can be specified using the _-f_ option. Non-existent folders will be created automatically.
 
-Mails can be marked as Unseen (new) using the _n_ option.
+Mails can be marked as Unseen (new) using the _-n_ option.
+
+Use option _--ignore-missing-date_ to ignore all messages with missing date. In this case, messages are imported to gmail with the creation date of the file.
```

Comment:
Shouldn't this be the "last modification time"?

Reply:
I thought about that too. But I think the date means the date when the message is sent. The creation date comes closer to that.
Or what do you think?

Reply:
But - there is no creation time ;-)

You are using getctime() in line 68, and that returns the change time on Linux:
https://docs.python.org/2/library/os.path.html

The wording should be different.

Reply:
Yes, you're right. For some reason it's tough to get the file creation date in python…
Do you have an idea for the wording?

Reply:
Just replace "creation time" with "last modification time"? Should be good enough.

Reply:
Done.